<launch>
  <node pkg="yac_ros" type="calib_mocap_capture_node" name="calib_mocap_capture_node" required="true" output="screen">
    <param name="imshow" value="true" />
    <param name="rosbag_path" value="/data/calib_mono.bag" />
    <param name="cam0_topic" value="/camera/color/image_raw" />
  </node>
</launch>
